react/w-books/src/app/screens/login/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import woloxLogo from 'app/assets/wolox.png';
+
+import styles from './styles.module.scss';
+
+interface InputsForm {
+  firstName: string;
+  lastName: string;
+  email: string;
+  password: string;
+  passwordConfirmation: string;
+}
+
+function Login() {
+  const { register, handleSubmit } = useForm<InputsForm>();
+
+  const onSubmit = (data: InputsForm) => {
+    const result = {
+      user: {
+        email: data.email,
+        password: data.password,
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        password_confirmation: data.passwordConfirmation,
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        first_name: data.firstName,
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        last_name: data.lastName,
+        locale: 'en'
+      }
+    };
+    console.log(result);
+  };
+
+  return (
+    <form className={`column ${styles.loginContainer}`} onSubmit={handleSubmit(onSubmit)}>
+      <img className={styles.mainLogo} src={woloxLogo} />
+      <label className={styles.label}>Nombre</label>
+      <input name="firstName" className={styles.input} ref={register} />
```

Comment:
all of this names could be placed on a constants file, also on the onSubmit function you could do like `email: data[USER_FIELDS.firstName]